projects/1.16.1/assets/a-v-a-in-minecraft/ava/lang/zh_cn.json

```
@@ -1,3 +1,185 @@
+    "item.ava.fn_fnc": "比利时 FN-FNC 自动步枪",
+    "item.ava.fn_fnc_dreamcatcher": "FN-FNC 捕梦网",
+    "item.ava.fn_fnc_fullmoon": "FN-FNC 祥云瑞月",
+    "item.ava.fn_fnc_magazine": "FN-FNC 弹匣",
+    "item.ava.xm8": "美制 XM8 轻型突击步枪",
+    "item.ava.xm8_frost": "XM8 急冻冰霜",
+    "item.ava.xm8_snowfall": "XM8 霜降冰晶",
+    "item.ava.xm8_magazine": "XM8 弹匣",
+    "item.ava.mp5sd5": "德制 MP5SD5 微声冲锋枪（固定式塑料枪托）",
+    "item.ava.mp5sd5_magazine": "MP5SD5 弹匣",
+    "item.ava.mk18": "美制 MK.18 MOD 0 突击步枪（特种部队用,M4A1的变种）",
+    "item.ava.mk18_air_warfare": "MK.18 Air Warfare",
+    "item.ava.mk18_kuyo_mon": "MK.18 九曜紋",
+    "item.ava.mk18_magazine": "MK.18 弹匣",
+    "item.ava.sr_25": "Knights SR-25狙击枪",
+    "item.ava.sr_25_knut": "Knut's M110 欢乐北极熊",
+    "item.ava.sr_25_magazine": "SR-25 弹匣",
+    "item.ava.fg42": "德制 FG42 伞兵步枪",
+    "item.ava.fg42_sumire": "FG42 侍魂",
+    "item.ava.fg42_dreamcatcher": "FG42 捕梦网",
+    "item.ava.fg42_magazine": "FG42 弹匣",
+    "item.ava.mp5k": "德制 MP5K 超短型冲锋枪",
+    "item.ava.mp5k_frost": "MP5K 急冻冰霜",
+    "item.ava.mp5k_magazine": "MP5K 弹匣",
+    "item.ava.m202": "M202 火箭发射器",
+    "item.ava.m202_rocket": "M202 火箭",
+    "item.ava.gm94": "GM-94 榴弹发射器",
+    "item.ava.gm94_grenade": "GM-94 榴弹",
+    "item.ava.sw1911_colt": "Colt SW1911 罪恶克星",
+    "item.ava.sw1911_magazine": "Colt SW1911 罪恶克星 弹匣",
+    "item.ava.python357": "Python.357 狂蟒",
+    "item.ava.python357_overrider": "Python OverRider 异域圣战",
+    "item.ava.python357_bullets": "Python.357 子弹",
+  
+    "item.ava.m67": "美制 M67 延时碎片式手榴弹",
+    "item.ava.mk3a2": "美制 MK3A2 进攻型手榴弹",
+    "item.ava.m116a1": "M116A1震!撼!弹!",
+    "item.ava.m18_grey": "M18烟雾弹 灰",
+    "item.ava.m18_purple": "M18烟雾弹 紫",
+    "item.ava.m18_toxic": "M18烟雾弹 毒",
+  
+    "item.ava.binocular": "望远镜",
+  
+    "item.ava.eu_standard_boots": "EU 基本靴子",
+    "item.ava.eu_standard_trousers": "EU 基本长裤",
+    "item.ava.eu_standard_kevlar": "EU 基本防弹衣",
+    "item.ava.eu_standard_helmet": "EU 基本头盔",
+    "item.ava.nrf_standard_boots": "NRF 基本靴子",
+    "item.ava.nrf_standard_trousers": "NRF 基本长裤",
+    "item.ava.nrf_standard_kevlar": "NRF 基本防弹衣",
+    "item.ava.nrf_standard_helmet": "NRF 基本头盔",
+  
+    "item.ava.ammo_kit": "子弹补给",
+    "item.ava.ammo_kit_ii": "子弹补给 II",
+  
+    "block.ava.gun_crafting_table": "AVA 工作台",
+    "block.ava.gun_colouring_table": "AVA 上色台",
+  
+    "block.ava.ammo_kit_supplier": "弹药补给器",
+  
+    "block.ava.attack_damage_boost_block": "攻击提升块",
+    "block.ava.health_boost_block": "生命提升块",
+  
+    "ava.aim": "瞄准",
+  
+    "ava.pull_grenade": "手榴弹插销拔出",
+    "ava.grenade_hit": "手榴弹撞击",
+  
+    "ava.bullet_flies_by": "子弹飞过",
+  
+    "ava.grenade_explodes": "手榴弹爆炸",
+    "ava.flash_grenade_explodes": "手榴弹爆炸",
+    "ava.rocket_explode": "火箭爆炸",
+    "ava.rocket_travel": "火箭飞过",
+    "ava.smoke_grenade_releases": "烟雾弹释放烟雾",
+  
+    "ava.night_vision_activate": "启用夜视鏡",
+    "ava.uav_captures": "望远镜抓取",
+    "ava.uav_captured": "你被敌人的UAV发現了!",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
    "ava.uav_captured": "你被敌人的UAV发现了!",
```
你是不是用的繁体转过来的啊，是的话请自检一边繁体字并全部替换。